### PR TITLE
Kuber predeployment

### PR DESCRIPTION
**Pull request by devtrap**
- Add kubernetes type as supported types for pre-deployment with kubernetesIacFiles key 
- Check if main kubernetes keys exist in kubernetes file

### CODE REVIEW

**Review by ajeybk — APPROVED** [review]
Looks good

File: src/processor/template_processor/base/base_template_constatns.py [hunk 1]
```diff
@@ -2,10 +2,13 @@
...
 from processor.template_processor.terraform_template_processor import TerraformTemplateProcessor
+from processor.template_processor.kubernetes_template_processor import KubernetesTemplateProcessor
+
 
 TEMPLATE_NODE_TYPES = {
     "cloudformation": AWSTemplateProcessor,
     "arm" : AzureTemplateProcessor,
     "deploymentmanager" : GoogleTemplateProcessor,
     "terraform" : TerraformTemplateProcessor,
+    "kubernetesIacFiles" : KubernetesTemplateProcessor
```
Comment (vatsalgit5118) [inline src/processor/template_processor/base/base_template_constatns.py]: **kubernetesIacFiles** is not a good name for snapshot type. Change it to **kubernetes**.

**Review by vatsalgit5118 — COMMENTED** [review]
Put the **kubernetes/__init__.py** file. 
Typo mistake while creating init file, **kubernetes/__inti__.py** .

File: src/processor/template_processor/base/base_template_constatns.py (hunk 1 above)
Comment (devtrap) [reply src/processor/template_processor/base/base_template_constatns.py]: kubernetes type will use for post-deployment ,I change this type to kubernetesObjectFiles

Comment (devtrap) [reply src/processor/template_processor/base/base_template_constatns.py:13]: init file exists in kubernetes folder!

**Comment by devtrap** [comment]
> Put the **kubernetes/**init**.py** file.
> Typo mistake while creating init file, **kubernetes/**inti**.py** .

init file exists in kubernetes folder!

File: src/processor/template_processor/base/base_template_constatns.py [hunk 2]
```diff
@@ -10,6 +11,7 @@
     "arm" : AzureTemplateProcessor,
     "deploymentmanager" : GoogleTemplateProcessor,
     "terraform" : TerraformTemplateProcessor,
+    "kubernetesObjectFiles" : KubernetesTemplateProcessor
```
Comment (vatsalgit5118) [inline src/processor/template_processor/base/base_template_constatns.py]: Add the comma(",") at the end.

Comment (devtrap) [reply src/processor/template_processor/base/base_template_constatns.py]: I added comma for resolve merging conflicts purpose

**Review by vatsalgit5118 — APPROVED** [review]

**Pull request merged**